pkg/deploy/api/v1/types.go

```diff
@@ -28,12 +28,17 @@ const (
 type DeploymentStrategy struct {
 	// Type is the name of a deployment strategy.
 	Type DeploymentStrategyType `json:"type,omitempty" description:"the name of a deployment strategy"`
+
 	// CustomParams are the input to the Custom deployment strategy.
 	CustomParams *CustomDeploymentStrategyParams `json:"customParams,omitempty" description:"input to the Custom deployment strategy"`
 	// RecreateParams are the input to the Recreate deployment strategy.
 	RecreateParams *RecreateDeploymentStrategyParams `json:"recreateParams,omitempty" description:"input to the Recreate deployment strategy"`
 	// RollingParams are the input to the Rolling deployment strategy.
 	RollingParams *RollingDeploymentStrategyParams `json:"rollingParams,omitempty" description:"input to the Rolling deployment strategy"`
+
+	// After is an optional set of actions to take after a deployment has completed.
+	After *AfterDeploymentActions `json:"after" description:"actions to take after a deployment has completed"`
```

Comment:
@smarterclayton not urgent, just want to make sure this suggestion isn't lost in the mix